packages/kbn-reporting/export_types_helpers_public/constants.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { LayoutParams } from '@kbn/screenshotting-plugin/common';
+import { Ensure, SerializableRecord } from '@kbn/utility-types';
+
+export const CSV_SEARCHSOURCE_IMMEDIATE_TYPE = 'csv_searchsource_immediate';
+// Licenses
+export const LICENSE_TYPE_TRIAL = 'trial' as const;
+
+export type IlmPolicyMigrationStatus = 'policy-not-found' | 'indices-not-managed-by-policy' | 'ok';
+
+export interface IlmPolicyStatusResponse {
+  status: IlmPolicyMigrationStatus;
+}
+
+/**
+ * @deprecated
+ */
+export type BaseParams = Ensure<
```

Comment:
I think the types and interfaces can be deleted from this file - they exist in the `types.ts` file